src/commands/optimize.ts

```diff
@@ -181,6 +189,26 @@ export default class Optimize extends Command {
     }]);
 
     this.optimizations = optimizationRes.optimization;
+    
+    if (!(this.optimizations?.length)) {
+      this.metricsMetadata.optimized = false;
+    }
+
+    // for (const optimizationSelected of optimizationRes.optimization) {
+    //   if (optimizationSelected.length) {
+    //     this.metricsMetadata[`optimization_${optimizationSelected}`] = true;
+    //   }
+    // }
```

Comment:
This simplification works correctly if we change the `value` pushed to `choices` as `removeComponents`, `moveToComponents` and `reuseComponents`(lines 169, 175 and 181). Otherwise, the metadata sent would be as follows:

![image](https://github.com/smoya/cli/assets/81691177/da3be36a-f2ec-4bfe-9a6c-d3657db7c0d3)

cc @smoya